src/webview_provider/queryResultPanel.ts

```diff
@@ -371,14 +371,9 @@ export class QueryResultPanel implements WebviewViewProvider {
         );
         return;
       }
-      window.showErrorMessage(
-        extendErrorWithSupportLinks(
-          "Encountered an unknown issue: " + exc.message + ".",
-        ),
-      );
```

Comment:
any particular reason to remove this??

Reply:
The thing is that dbt cloud doesn't have PythonExceptions, this code path was never really supposed to hit before but now it is every dbt cloud error.